Update readme file (#186)
samg/timetrap: README.md

This will place a ``t`` executable in your path.

> If you have errors while parsing the documentation, use `--no-document` option when installing the gem, or other option is to `gem install rdoc` before installing the `timetrap`. This is a known issue from [rdoc](https://github.com/ruby/rdoc/commit/5f9603f35d8e520c761015810c005e4a5beb97c3)

Reviewer: Just a couple of nits:

```suggestion
If you run into errors while parsing the documentation, use the `--no-document` option when installing the gem. The other option is to run `gem install rdoc` before installing the `timetrap` gem. This is a known issue from [rdoc](https://github.com/ruby/rdoc/commit/5f9603f35d8e520c761015810c005e4a5beb97c3)
```
